### DISCUSSION TITLE
Perfect Seed Number

### DISCUSSION THREAD

**Discussion by riazuddinshaik** [lllyasviel/Fooocus#3241] [Q&A] [answered]
@lllyasviel Hi. Thankyou for awesome work. Could you please let me know what is the best seed number which works best for image generation (inpaint feature)? I tried random and also 404 and 8545904197726069955....... but i'm not satisfied

Could you let me know plz?

**Comment by mashb1t** [comment, marked as answer]
=> https://github.com/lllyasviel/Fooocus/issues/3242#issuecomment-2210281970
